iceoryx_posh/CMakeLists.txt

```diff
@@ -48,12 +48,6 @@ set(${PROJECT_NAME}_DIR ${CMAKE_CURRENT_LIST_DIR}/cmake
 #
 ########## posh core lib ##########
 #
-# If requested, enable 1:n communication only
-if(one-to-many)
```

Comment:
is this intentionally removed?

Reply:
Yes! It is now handled directly via `iceoryx_posh_types.hpp`. As it turns out it suffices if we declare there an alias to the correct type and a constexpr to the queue type like we did for the sender and receiver ports

Reply:
@mossmaurice please elaborate!

Reply:
Will push @elBoberido and my solution soon on #292